internal/conf/rate.go

```diff
@@ -45,6 +51,7 @@ func (r *Rate) Decode(value string) error {
 		return fmt.Errorf("rate: over-time part of rate value %q failed to parse as duration: %w", value, err)
 	}
 
+	r.typ = BurstRateType
```

Comment:
just to clarify, this means that any of the existing rate limits that follow the notation `X/Y` will now be interpreted as `<burst>/<rate>` while before, it was interpreted as `<quota>/<interval>` (e.g. 0/1h) ? 

if we happen to have a project with an existing setting of 100/24hr, then it would be interpreted as a burst limiter - will there be a change in behaviour here?

Reply:
Currently projects only use the float format,  the only ones which use the burst format are the email restricted projects. Though I suppose it could change self hosted projects behavior, they would only be able to send 1 email per 24 hours after the initial 100 were sent.

A previous version of this I used the left hand side as the number of events to generate per the right hand side. The hope of that was to be less impactful to the old format. It was decided it wasn't used and the simplicity of the <burst> <events to generate per second> was better.